dom-manipulation/main.js

```diff
@@ -0,0 +1,22 @@
+var numberOfClick = 0;
+var buttonishot = document.querySelector('.hot-button');
+var clickcounter = document.querySelector('.click-count');
+function handleClick(event) {
+  numberOfClick++;
+  console.log(numberOfClick);
+  clickcounter.textContent;
+  if (numberOfClick.className < 4) {
```

Comment:
number of click is a number, so it doesn't have a classname, and what you should be checking is the number of clicks, that is a numeric value which indicates the number of times clicked